products/ceqr_survey/models/intermediate/_intermediate_models.yml

```diff
@@ -39,54 +39,79 @@ models:
   - name: int__elevated_railways
     description: Buffered geometries for elevated/exposed railways
     columns:
-      - name: variable
+      - name: variable_type
         test:
           - not_null
-      - name: id
+      - name: variable_id
         test:
           - not_null
-      - name: geom
+      - name: raw_geom
+        tests:
+          - not_null
+      - name: buffer
         tests:
           - not_null
 
   - name: int__nysdec_state_facility_permits
     description: Buffered geometries for state facility permits
     columns:
-      - name: variable
+      - name: variable_type
         test:
           - not_null
-      - name: id
+      - name: variable_id
         test:
           - not_null
-          - unique
```

Comment:
Noticed that you removed a test for uniqueness for `variable_id` for a few tables... Are they not?

Reply:
Ah. copy-pasta. Added them back.